pages/common/pigz.md

```diff
@@ -0,0 +1,23 @@
+# pigz
+
+> Multithreaded zlib compression utility.
+
+- Compress a file with default options:
+
+`pigz {{filename.ext}}`
+
+- Compress a file using compression method [1-9] and number of processors [-pN]:
```

Comment:
I read that you can also specify `-0` and `-11`, so it might not be wise to limit it in your explanation. 

You should be able to drop the `[-pN]` from your description I think, the example clarifies as much.